index.html

```diff
@@ -35,14 +35,13 @@
   <nav class="navbar" id="divNav">
     <div class="flex-nav">
       <div class="nav-header">
-        <h2><span>Way To</span> Health</h2>
+        <h2 id="nav-logo"><span>Way To </span>Health</h2>
```

Comment:
Makes it easier to target specifically this element for the use of styling of the logo and not affecting other H2s on the page should they be used